normandy/classifier/api.py

```diff
@@ -6,7 +6,8 @@
 
 
 class FetchBundle(views.APIView):
-
+    authentication_classes = []
+    permission_classes = []
```

Comment:
To be clear, we can make the API specifically CSRF-free, without making that the default for all views.